plugins/wacom-raw/wacom-raw.quirk

```diff
@@ -0,0 +1,35 @@
+# Devices that do "replug" and thus don't change VID:PID to the bootloader
+# need to have an extra GUID of WacomAES or WacomEMR added so that the flash
+# constants are set correctly.
+
+# Dell XPS-15
```

Comment:
You should probably expand this smbios string in comment. XPS 15 releases quite regularly and the panels will vary model to model.

Reply:
I believe it should be XPS 9575

Reply:
Indeed it is, thanks.